example/rlhf/demo_rl.ipynb

```diff
@@ -302,62 +311,19 @@
+      "\u001b[0;31mAttributeError\u001b[0m: 'RLHFConfig' object has no attribute 'total_ppo_epochs'"
```

Comment:
It seems that there is still an error here. Can you clean up the output and write a to do note?

Reply:
Alternatively, can we remove this step 3 rl notebook example? For step 3, we use the py file, not the notebook file, as the example. 
In the mlflow folder, I removed the notebook for step 3 rl.

Reply:
Error commented. Added a to-do comment in that block.